src/com/android/volley/toolbox/DiskBasedCache.java

```diff
@@ -440,7 +441,7 @@ public CacheHeader(String key, Entry entry) {
          * @param is The InputStream to read from.
          * @throws IOException
          */
-        public static CacheHeader readHeader(InputStream is) throws IOException {
+        public static CacheHeader readHeader(InputStream is, boolean readFull) throws IOException {
```

Comment:
Could rename `readFull` to be more descriptive like `readHeaders`, `includeHeaders', `saveHeaders` etc